lldb/tools/lldb-rpc-gen/RPCCommon.cpp

```diff
@@ -118,6 +118,7 @@ static constexpr llvm::StringRef MethodsWithPointerPlusLen[] = {
     "_ZN4lldb8SBTarget15GetInstructionsEyPKvm",
     "_ZN4lldb8SBTarget25GetInstructionsWithFlavorEyPKcPKvm",
     "_ZN4lldb8SBThread18GetStopDescriptionEPcm",
+    "_ZNK4lldb8SBThread18GetStopDescriptionERNS_8SBStreamE",
```

Comment:
Remind me why this goes here. Something like "Streams are represented in RPC as a buffer and a length"?

_ZN4lldb8SBThread18GetStopDescriptionEPcm is literally that, so I wonder if this new thing in fact doesn't need to be here.